modules/friendlytalkback.js

```diff
@@ -85,27 +85,24 @@ Twinkle.talkback.callback = function() {
 	result.tbtarget[0].dispatchEvent(evt);
 
 	// Check whether the user has opted out from talkback
-	var query = {
-		action: 'query',
-		prop: 'extlinks',
-		titles: 'User talk:' + mw.config.get('wgRelevantUserName'),
-		elquery: 'userjs.invalid/noTalkback',
-		ellimit: '1',
-		format: 'json'
-	};
-	var wpapi = new Morebits.wiki.api('Fetching talkback opt-out status', query, Twinkle.talkback.callback.optoutStatus);
-	wpapi.post();
+	var user = new Morebits.wiki.user(mw.config.get('wgRelevantUserName'), 'Fetching talkback opt-out status');
+	user.setNotifySkips('userjs.invalid/noTalkback', Twinkle.optoutTemplates);
```

Comment:
Is it worth restructuring talkback/{{notalkback}} to use the new optout method/link?  It's not quite correct since we're not using it to optout of receiving a message, but then again we're not using this to send a message anyway.  In theory, others could use {{notalkback}}, so no, but then again I don't think anybody actually does.